conf/homeserver.yaml

```diff
@@ -1342,16 +1342,17 @@ password_providers:
    - module: "ldap_auth_provider.LdapAuthProvider"
      config:
        enabled: true
+       mode: "search"
        uri: "ldap://localhost"
        start_tls: false
        base: "ou=users,dc=yunohost,dc=org"
        attributes:
           uid: "uid"
           mail: "mail"
           name: "givenName"
-#        #bind_dn:
+#        #bind_dn: # TODO Add authentication to have the filter working
```

Comment:
Because, as documented [here](https://github.com/matrix-org/matrix-synapse-ldap3#usage), there are 2 solution to authenticate:

- simple: `bind_dn` and `bind_password` are not mandatory but the filter don't work.
- search: `bind_dn` and `bind_password` are mandatory and the filter will work.

Reply:
Whokay